src/console/autoSwitcher.ts

```diff
@@ -1,62 +1,77 @@
 import { EventEmitter } from "events";
-import OBSWebSocket from "obs-websocket-js";
+import { isIPv6 } from "net";
```

Comment:
If this function exists (as well as `isIPv4`) in the built in `net` package, how come we need to install the `is-ip` package?